src/PhpDocParser/PhpParser/SmartPhpParserFactory.php

```diff
@@ -42,8 +43,9 @@ private function createPHPStanParser(Parser $parser): CachedParser
         $nameResolver = new NameResolver();
         $variadicMethodsVisitor = new VariadicMethodsVisitor();
         $variadicFunctionsVisitor = new VariadicFunctionsVisitor();
+        $propertyHookNameVisitor = new PropertyHookNameVisitor();
 
-        $simpleParser = new SimpleParser($parser, $nameResolver, $variadicMethodsVisitor, $variadicFunctionsVisitor);
+        $simpleParser = new SimpleParser($parser, $nameResolver, $variadicMethodsVisitor, $variadicFunctionsVisitor, $propertyHookNameVisitor);
```

Comment:
@TomasVotruba we need immediate release due to new required fifth parameter on PHPStan `SimpleParser` that pass `PropertyHookNameVisitor`

Reply:
Just released Rector 2.0.5 due to this required parameter in PHPStan 2.1.0 

https://github.com/rectorphp/rector/releases/tag/2.0.5